### PR TITLE
Add support for fraud analysis.

### PR DESCRIPTION
**Pull request by kassio**
This adds fraud analysis support regarding [braspag documentation](http://apidocs.braspag.com.br/).

### CODE REVIEW

File: lib/braspag-rest/hashie/iutrash.rb [hunk 1]
```diff
@@ -5,14 +5,27 @@ class IUTrash < Hashie::IUDash
...
+        unless value.nil?
+          attributes[from] = value
         end
       end
     end
+
+    private
+
+    def nested_inverse(value)
+      if [Array, Set].include?(value.class)
```
Comment (bvicenzo) [inline lib/braspag-rest/hashie/iutrash.rb]: Wdyt about to check if value respond to map ?
```rb
if value.respond_to?(:map)
```

Comment (kassio) [inline lib/braspag-rest/hashie/iutrash.rb]: I'll do it.

**Comment by bvicenzo** [comment]
👏 👏 🐑

File: lib/braspag-rest/hashie/iutrash.rb (hunk 1 above)
Comment (kassio) [inline lib/braspag-rest/hashie/iutrash.rb]: Actually, this isn't good, because `Hash` also responds to `map`. 😊

Comment (bvicenzo) [inline lib/braspag-rest/hashie/iutrash.rb]: That's true 😊

**Comment by danielcoca** [comment]
👏 👏  :shipit:

**Comment by LuciannaAlmeida** [comment]
:+1: :shipit:

**Comment by marciotoshio** [comment]
👍

**Pull request merged**